src/main/java/pucsp/mycash/model/BankStatement.java

```diff
@@ -0,0 +1,17 @@
+package pucsp.mycash.model;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name="bank_statement")
+public class BankStatement {
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    private Long id;
+    private float incomes;
+    private float expenses;
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User userId;
```

Comment:
Add `@NotNull`
Add `@JsonManagedReference`
```suggestion
    private User user;
```